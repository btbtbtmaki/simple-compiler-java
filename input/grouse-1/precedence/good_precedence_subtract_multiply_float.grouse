main {

  print 7.5 - 1.2 * 8.1 nl;
  print 5.1 - 8.3 * 9.0 nl;
  print 1.8 - 1.5 * 7.8 nl;
  print 1.6 - 3.5 * 3.6 nl;
  print 7.0 - 6.1 * 5.3 nl;
  print 4.3 - 4.9 * 6.3 nl;
  print 8.8 - 1.5 * 7.6 nl;
  print 9.1 - 4.2 * 9.7 nl;
  print 8.1 - 9.2 * 9.6 nl;
  print 4.9 - 6.3 * 4.1 nl;
  print 7.5 - 6.8 * 2.6 nl;
  print 9.4 - 7.7 * 5.1 nl;

  print 4.4 * 4.2 - 5.6 nl;
  print 1.8 * 2.0 - 1.3 nl;
  print 6.1 * 8.3 - 9.5 nl;
  print 4.9 * 1.2 - 7.7 nl;
  print 8.0 * 9.2 - 9.6 nl;
  print 3.1 * 3.5 - 5.6 nl;
  print 5.5 * 7.4 - 3.8 nl;
  print 9.7 * 2.4 - 7.1 nl;
  print 1.4 * 5.7 - 3.0 nl;
  print 5.8 * 2.8 - 1.0 nl;
  print 6.2 * 5.1 - 7.1 nl;
  print 9.7 * 8.4 - 2.7 nl;

  print 5.3 - 8.7 * 7.1 - 1.9 nl;
  print 1.4 - 2.1 * 6.5 - 7.5 nl;
  print 7.4 - 2.7 * 8.3 - 3.3 nl;
  print 2.6 - 5.2 * 8.1 - 7.1 nl;
  print 4.3 - 2.3 * 6.2 - 9.1 nl;
  print 3.5 - 4.6 * 1.7 - 6.5 nl;
  print 3.3 - 3.3 * 5.9 - 8.9 nl;
  print 7.3 - 9.8 * 7.7 - 9.4 nl;
  print 9.3 - 8.1 * 5.5 - 9.8 nl;
  print 2.8 - 5.2 * 5.0 - 6.6 nl;
  print 3.3 - 6.5 * 3.1 - 2.8 nl;
  print 9.8 - 5.5 * 6.5 - 1.7 nl;

  print 7.3 * 7.6 - 5.2 * 7.0 nl;
  print 4.4 * 5.2 - 1.3 * 4.5 nl;
  print 3.8 * 5.8 - 4.4 * 2.5 nl;
  print 5.5 * 1.0 - 8.2 * 5.4 nl;
  print 6.3 * 5.2 - 2.7 * 8.9 nl;
  print 1.9 * 3.3 - 5.6 * 9.9 nl;
  print 9.8 * 5.8 - 1.7 * 2.0 nl;
  print 9.5 * 6.0 - 3.0 * 1.9 nl;
  print 2.4 * 2.7 - 5.9 * 2.3 nl;
  print 3.1 * 8.5 - 8.9 * 9.5 nl;
  print 4.1 * 3.5 - 7.1 * 2.0 nl;
  print 8.2 * 2.6 - 7.7 * 4.1 nl;

}

