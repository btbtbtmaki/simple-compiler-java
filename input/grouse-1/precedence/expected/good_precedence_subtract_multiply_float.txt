-2.22
-69.6
-9.9
-11
-25.33
-26.57
-2.6
-31.64
-80.22
-20.93
-10.18
-29.87
12.88
2.3
41.13
-1.82
64
5.25
36.9
16.18
4.98
15.24
24.52
78.78
-58.37
-19.75
-18.31
-46.62
-19.06
-10.82
-25.07
-77.56
-45.05
-29.8
-19.65
-27.65
19.08
17.03
11.04
-38.78
8.73
-49.17
53.44
51.3
-7.09
-58.2
0.15
-10.25
